www/js/app.js

```diff
@@ -1248,8 +1249,9 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             // We have to remove the focus from the search field,
             // so that the keyboard does not stay above the message
             $("#searchArticles").focus();
-            alert("Archive not set : please select an archive");
-            $("#btnConfigure").click();
+            uiUtil.systemAlert("Alert", "Archive not set : please select an archive", false).then(function () {
+                $("#btnConfigure").click();
```

Comment:
Is it necessary to await a response before doing the btConfigure click? I think it might be feasible for this to run in the background behind the modal screen, so that the user will be on the Configuration screen already when they exit the modal. But I can see there are pros and cons, so maybe we can ask mossroy's opinion of this.

Reply:
Yes, we can do this either way. 

In my opinion, I think we should await this as once the user cancels the modal, then it'll give a feel of getting redirected to the place where they need to take the necessary action. Otherwise, as soon as someone clicks on the search button without loading the ZIM, the modal will fire and the page will change in the background which might confuse the user.....

Reply:
I tend to agree with @gaurav7019 
It's also certainly safer : `alert` function was a real modal window, blocking the UI.
The bootstrap popup is not modal, so we should avoid doing things in the background (that could for example try to open another bootstrap popup...)